core/src/main/scala/com/raphtory/internals/storage/pojograph/entities/internal/PojoVertex.scala

```diff
@@ -10,34 +10,47 @@ import scala.collection.mutable
 class PojoVertex(msgTime: Long, val vertexId: Long, initialValue: Boolean)
         extends PojoEntity(msgTime, initialValue) {
 
-  var incomingEdges: mutable.Map[Long, PojoEdge] =
+  private val intoE: mutable.Map[Long, PojoEdge] =
```

Comment:
Just a preference for shorter var names? If we are changing, would prefer `inE` as into somewhat sounds like it's something actually inside of an edge.